vscode-extension/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "relay-lsp-client",
+  "displayName": "Relay for VSCode",
```

Comment:
I think the fact that it's in the VSCode Marketplace is enough to communicate that it's for VSCode. I'll chat with the team about what we want here. I'm thinking "Relay GraphQL". Fine to merge this as is and update once we decide.